source/open-telemetry/tests/transaction/transaction.yml

```diff
@@ -0,0 +1,117 @@
+        enableCommandPayload: true
+  - database:
+      id: database0
+      client: client0
+      databaseName: transaction-tests
+  - collection:
+      id: collection0
+      database: database0
+      collectionName: test
+  - session:
+      id: session0
+      client: client0
+initialData:
+  - collectionName: test
+    databaseName: transaction-tests
+    documents: []
+tests:
+  - description: observeTracingMessages around transaction
+    operations:
+      - object: session0
+        name: startTransaction
+      - object: collection0
+        name: insertOne
+        arguments:
+          session: session0
+          document:
+            _id: 1
+      - object: session0
+        name: commitTransaction
+      - name: find
+        object: collection0
+        arguments:
+          filter:
+            x: 1
+    expectTracingMessages:
+      client: client0
+      ignoreExtraSpans: false
+      spans:
+        - name: transaction
+          tags:
+            db.system: mongodb
+          nested:
+            - name: insert transaction-tests.test
+              tags:
+                db.system: mongodb
+                db.namespace: transaction-tests
+                db.collection.name: test
+                db.operation.name: insert
+                db.operation.summary: insert transaction-tests.test
+              nested:
+                - name: command insert
+                  tags:
+                    db.system: mongodb
+                    db.namespace: transaction-tests
+                    server.address: { $$type: string }
+                    server.port: { $$type: ['long', 'string'] }
+                    server.type: { $$type: string }
+                    db.query.summary: insert
+                    db.query.text:
+                      $$matchAsDocument:
+                        $$matchAsRoot:
+                          insert: test
+                          ordered: true
+                          $db: transaction-tests
+                          lsid: { $$sessionLsid: session0 }
+                          txnNumber: 1
+                          startTransaction: true
+                          autocommit: false
+                          documents:
+                            - _id: 1
+                    db.mongodb.lsid: { $$sessionLsid: session0 }
+            - name: commitTransaction admin.$cmd
+              tags:
+                db.system: mongodb
+              nested:
+                - name: command commitTransaction
+                  tags:
+                    db.system: mongodb
+                    db.query.text:
+                      $$matchAsDocument:
+                        $$matchAsRoot:
+                            commitTransaction: 1
+                            $db: admin
+                            lsid: { $$sessionLsid: session0 }
+                            txnNumber: 1
+                            autocommit: false
+        - name: find transaction-tests.test
+          tags: {}
```

Comment:
Why is `tags` empty here? If you don't mean to assert any tags would you not just omit the field entirely?

This might be a sensible place to leave a YAML comment explaining why you're not bothering to assert something (even if you remove this line entirely).